packages/components/src/components/hds/popover-primitive/index.hbs

```diff
@@ -0,0 +1,44 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class={{this.classNames}} ...attributes {{this.setupPrimitiveContainer}}>
+  {{#if this.toggleContainsInteractive}}
+    <div
+      class="hds-popover-primitive__toggle"
+      aria-controls={{this.popoverId}}
```

Comment:
https://www.w3.org/TR/wai-aria-1.2/#aria-controls

This `div` won't ever be able to control the popover content because it's not a control (interactive). 

Is there a way for you to forward these attributes to the yielded interactive element?

Reply:
> https://www.w3.org/TR/wai-aria-1.2/#aria-controls
> This `div` won't ever be able to control the popover content because it's not a control (interactive).

Ok, so I have to remove it?

> Is there a way for you to forward these attributes to the yielded interactive element? Perhaps a helper?

As mentioned in other conversations, no it's not possible.

Reply:
When I look at the flyout's `.hbs` file, I see that we're forwarding the `id` attribute inside of the yield's hash. I think we could try a similar approach here, WDYT?

Reply:
well, we already know the value will be `{{this.popoverId}}` for the relevant `aria-` attributes, yes? They wouldn't have to do anything, I wouldn't think? 


But we do have a concise list of what could be the trigger, from the API: 

> "Invoking attributes" are only supported on buttons (see [https://open-ui.org/components/popover.research.explainer](https://open-ui.org/components/popover.research.explainer/#:~:text=Invoking%20attributes%20only%20supported%20on%20buttons)). From what I can tell, this is what that means:
`<button type=[button|submit|reset]>` (if the button would not otherwise submit a form)
`<input type=[button|submit|reset|image]>` (if it would not otherwise submit a form)
`<input type=[text|email|password|search|tel|url]>` (activated by down arrow only)

Reply:
The toggle works in two ways:
- it can be activated via “soft” events (hover/focus) using the native JS methods [`showPopover`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/showPopover), [`hidePopover`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidePopover), [`togglePopover`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/togglePopover)
    - https://github.com/hashicorp/design-system/blob/03-popover-primitive-implementation/packages/components/src/components/hds/popover-primitive/index.js#L77-L83
    - https://github.com/hashicorp/design-system/blob/03-popover-primitive-implementation/packages/components/src/components/hds/popover-primitive/index.js#L242-L255
- it can be activated via”click” events using the native [`popovertarget`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#popovertarget) attribute (we assign it via JS to the `<button class="hds-rich-tooltip__toggle">`)
   - https://github.com/hashicorp/design-system/blob/03-popover-primitive-implementation/packages/components/src/components/hds/popover-primitive/index.js#L99-L110

The first way ("soft" events) is the default one (and preferred one) in the `RichTooltip` component; for that case, any element can be used as content of the "toggle" container (interactive as well as non-interactive elements).